chains/orchestrator-paras/node/src/cli.rs

```diff
@@ -68,7 +68,9 @@ pub enum Subcommand {
     SoloChain(SoloChainCmd),
```

Comment:
What does this mean? Why would orchestrator para start solochain?

Reply:
Or is this embedded node?

Reply:
Yeah as discussed on Slack I found this terminology confusing. "orchestrator paras" is run by collators while "orchestrator relay" is run by validators. So even with dancelight runtime we run "orchestrator paras" node. Note that it is already like that on master.